src/index.js

```diff
@@ -82,6 +82,10 @@ export default class Voices {
     return this.byLangCode('pl-PL')
   }
 
+  spanish () {
+    return this.byLang('Castilian Spanish')
```

Comment:
should be only `Spanish` here, not `Castilian Spanish`. Because it's Spanish in general and must return all Spanish variants.

Reply:
That's why the test is failing